frontend/index.html

```diff
@@ -10,13 +10,15 @@ <h1>Mars Rover Project</h1>
 <h1 style="color: blue">Welcome...</h1>
 <image class="Curiosity Image" src="PIA20602_small.jpg" alt="Curiosity self portrait">
   <p><em>Self portrait of Curiosity Rover at Okoruso Drill Hole</em></p>
+   <p> Total Sols Days for Curiosity Rover: <strong id="sol-days"></strong></p>
   <section id="time-section">
  <p> Your local date and time is: </p> 
   <em id="local-date"></em> 
    <p> Kennedy Space Centre current date and time is: </p> 
    <em id="kennedy-space-date"></em>
  <p> Your local timezone is : </p> 
  <strong id="difference"></strong>
+
```

Comment:
Let's continue keeping things nice and tidy :)

Can we make sure there are no empty lines that aren't intentional, and also to make sure everything is indented correctly?